app/src/components/integrations/modals/addIntegrationModal/addIntegrationModal.jsx

```diff
@@ -79,8 +79,12 @@ const AddIntegrationModal = ({ data, initialize, change, handleSubmit, dirty })
   };
 
   const onSubmit = (newData) => {
-    if (isGlobal && !customProps.editAuthMode) {
-      trackEvent(PLUGINS_PAGE_EVENTS.clickCreateGlobalIntegration(data.instanceType));
+    if (isGlobal) {
+      if (!customProps.editAuthMode) {
+        trackEvent(PLUGINS_PAGE_EVENTS.clickCreateGlobalIntegration(data.instanceType));
+      } else {
+        trackEvent(PLUGINS_PAGE_EVENTS.clickEditGlobalIntegration(data.instanceType));
```

Comment:
Does this change related to the task?

Reply:
Yes, Yuliya said that this is part of EPMRPP-88953 task.